tests/CMakeLists.txt

```diff
@@ -1,17 +1,19 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 project(GSLTests CXX)
+enable_testing()  # again, for support standalone testing
```

Comment:
Only add the enable_testing() if GSLTests is the top level project.

See the main CMakeLists.txt on how to check if a project is top level.

Also the documentation style is inconsistent with your others. Move the comment above enable_testing instead of to the side.

Reply:
> Only add the enable_testing() if GSLTests is the top level project.
> See the main CMakeLists.txt on how to check if a project is top level.

Do you suggest to relocate the line under `if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)` condition?

> Also the documentation style is inconsistent with your others. Move the comment above enable_testing instead of to the side.

The comment is short enough. I do not think there is much point in splitting.

Reply:
"Do you suggest to relocate the line under if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR) condition?"

Yes that's correct.

Reply:
I agree with @personWhoWritesCmakeExamples it's inconsistent.
But it's a minor style issue.

The main thing for me is just making sure the rational is clear.

```cmake
project(GSLTests 
    LANGUAGES CXX
) 

if (${CMAKE_CURRENT_SOURCE_DIR} STREQUAL ${CMAKE_SOURCE_DIR})
    # Enable standalone testing.
    enable_testing()
endif()
```